workloads/workload.yml

```diff
@@ -42,8 +42,8 @@
   - name: "Including cluster variables..."
     include_vars: "{{ item }}"
     loop:
-      - "../{{ ocp_version | mandatory }}.x/my_vars.yml"
-      - "../{{ ocp_version }}.x/ocp{{ ocp_version }}_vars.yml"
+      - "{{ my_vars_dir | mandatory }}/my_vars.yml"
```

Comment:
@fbladilo do you want to make this var mandatory?

Reply:
@pranavgaikwad It has to be set, the same variable now contains the default location for my_vars.yml directory (../<OCP_VERSION>.x) or whaterver is optionally set by "-m".

Reply:
understood